bump_dev_version.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
Maybe put this script in ``.ci`` folder, to not further fill the root folder?

Reply:
ok